fizzbuzz.rb

```diff
@@ -10,3 +10,12 @@
 #   otherwise
 #       return the original number (as a string)
 
+def fizz_buzz(number)
+  if(number == 3)
```

Comment:
**issue:** The number should be divisible with three, so the function works for values like 6,9,etc. 
```suggestion
  if(number % 3 == 0)
```

Reply:
Ohh that makes sense, thank you for providing some different test input — this will be useful